spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -225,7 +211,51 @@ data class SpdxLicenseExceptionExpression(
     override fun toString() = id
 }
 
-sealed class SpdxSimpleExpression : SpdxExpression()
+sealed class SpdxSingleLicenseExpression : SpdxExpression()
```

Comment:
Would it make sense to add KDoc for the new types?

Reply:
Done.